frontend/src/Routes/routes.tsx

```diff
@@ -22,6 +25,9 @@ const Main: React.FC = () => {
       <Route path="/dashboard" element={<Dashboard />} />
       <Route path="/admin/users-list" element={<AdminUserPage />} />
       <Route path="/student/category/:id/" element={<LessonAnswerPage />} />
+      <Route path="/admin/dashboard" element={<AdminDashboard/>} />
+      <Route path="/admin/categories" element={<CategoryListPage/>} />
+      <Route path="/admin/category/:id/addword" element={<AddWordPage/>} />
```

Comment:
use kebab case for routes

```suggestion
      <Route path="/admin/category/:id/add-word" element={<AddWordPage/>} />
```